### DISCUSSION TITLE
[Test] Local test using Google Storage

### DISCUSSION THREAD

**Discussion by SalvadorCali** [mia-platform/community#99] [Q&A] [answered]
Hi everyone!

We need to do some integration tests on one of our services that reads and parses csv files from Google Storage and send messages to Kafka.
Do you know any docker images for Google Storage or have any advices?
I only found this one: https://github.com/fsouza/fake-gcs-server

**Comment by danibix95** [comment, marked as answer]
Hi @SalvadorCali,

The image you mentioned can be used to achieve your use case.
We combined it with [Testcontainers](https://www.testcontainers.org/) in some of our services' tests, exploiting the `GenericContainer` feature. Testcontainers supports multiple languages, so that you should be able to easily integrate them in your project.

**Reply by SalvadorCali** [reply]
Thank you!
